mantle/platform/api/gcloud/compute.go

```diff
@@ -33,8 +34,56 @@ func (a *API) vmname() string {
 	return fmt.Sprintf("%s-%x", a.options.BaseName, b)
 }
 
+// ["local-ssd:interface=NVME,size=375"]
+func ParseDiskSpec(spec string, zone string) (*compute.AttachedDisk, error) {
```

Comment:
Optional, but I think it'd be pretty easy to have a unit test or two for this function.